### ISSUE TITLE
Regular expressions with variables in cypher queries

### ISSUE THREAD

**Issue by Kabenla** [SciGraph/SciGraph#227]
Is it possible to do something like
 query: |
            MATCH (n)
            where n.label =~ .\*{term}.\*
            RETURN distinct n

Where {term} is a variable in the run configuration yaml file?

**Comment by cmungall** [comment]
Hi @Kablena - SciGraph is intended as a layer over Neo4J (although it does allow pass-through of cypher queries). Your question is about general cypher queries and might be best found on Neo4J lists/stackoverflow etc.

If you are using SciGraph services, paths such as https://scigraph-ontology.monarchinitiative.org/scigraph/docs/#!/vocabulary/searchByTerm will utilize lucene in searches, which satisfies your .*{term}.* case, but we don't have support for regexes in general. If you think this is useful we could look at adding, if it proves efficient.

**Comment by Kabenla** [comment]
Hi, thanks for gettign back.  It is not critical that we have the regular expressions.  I used CONTAINS instead and it works pretty well.  I will close issue

**Comment by kshefchek** [comment]
@Kabenla is the issue that regular expressions do now work in the dynamic services portion of the run config yaml?

**Comment by Kabenla** [comment]
Yes that is the issue.  The application exceptions when I attempt to use regular expressions.
